app/assets/javascripts/common/reports.js

```diff
@@ -1693,15 +1695,25 @@ function baseBarChartConfig() {
 const dynamicChartSegementDashed = (
   ctx,
   numberOfXAxisTicks,
-  numberOfDashedSegments = 1
+  endDateKey,
+  monthsDashedBase = 1,
 ) => {
-  const dashStyle = [4, 3]
-  const segmentStartIndex = ctx.p0DataIndex
-  return isSegmentDashed(segmentStartIndex, numberOfXAxisTicks, numberOfDashedSegments) ? dashStyle : undefined;
+
+  const endDateKeyIndex = monthIndexFromDateKey(endDateKey);
+  const dateNowMonth = new Date().getMonth()
+
+  const monthsInPast = dateNowMonth - endDateKeyIndex;
+  const dashedSegments = Math.max(0, monthsDashedBase - monthsInPast);
+
+  const segmentStartNodeIndex = ctx.p0DataIndex
```

Comment:
What does `ctx.p0DataIndex` return?

Reply:
Chartjs looks at each segment one by one starting on the left and moving to the right 
Each segment has a p0 and a p1 index which are the nodes at each end of the segment (left and right) 
Ctx.p0DataIndex Is the index 0 node the node on the left of the segment